--- a/sesman/env.c      2022-04-19 10:52:33.572142092 -0400
+++ b/sesman/env.c      2022-04-12 09:03:12.314596201 -0400
@@ -150,7 +150,6 @@
             /* pulse source socket */
             g_snprintf(text, sizeof(text) - 1, CHANSRV_PORT_IN_BASE_STR, display);
             g_setenv("XRDP_PULSE_SOURCE_SOCKET", text, 1);
-            g_setenv("REMOTE_CLIENT_IP", text, 1);
             if ((env_names != 0) && (env_values != 0) &&
                     (env_names->count == env_values->count))
             {
